cypress/support/fragments/bulk-edit/bulk-edit-actions.js

```diff
@@ -922,4 +936,9 @@ export default {
       Option({ value: 'SUPPRESS_FROM_DISCOVERY' }).exists(),
     ]);
   },
+
+  fillLocation(location) {
+    cy.do(Button('Select control\nSelect location').click());
+    cy.get('input[aria-label=" options filter"]').type(location).type('{enter}');
```

Comment:
Can be done with SelectionList interactor
const locationList = SelectionList(...)
cy.do(locationList.filter(location))